app/assets/stylesheets/forever_style_guide/modules/_hero_simple.scss

```diff
@@ -1,16 +1,27 @@
+$screen-almost_sm: 580px; // between xs -> sm breakpoints
```

Comment:
Does this have more global use? Could it be added to breakpoints.scss? Maybe $screen-sm_to_md?